package.json

```diff
@@ -128,10 +128,7 @@
   },
   "dependencies": {
     "@twilio/voice-errors": "1.7.0",
-    "@types/md5": "2.3.2",
     "events": "3.3.0",
-    "loglevel": "1.6.7",
-    "md5": "2.3.0",
```

Comment:
We're removing md5 here, is it because we both missed removing it in the other PR?